sklearn/manifold/t_sne.py

```diff
@@ -518,15 +518,27 @@ class TSNE(BaseEstimator):
         learning rate is too low, most points may look compressed in a dense
         cloud with few outliers. If the cost function gets stuck in a bad local
         minimum increasing the learning rate may help.
+        Some discussion on how to set learning rate optimally can be found
+        at https://doi.org/10.1101/451690. Effective use of this parameter has
```

Comment:
Use the References section and ReST citation format instead.

Reply:
> Use the References section and ReST citation format instead.